docs/logs/getting-started/README.md

```diff
@@ -51,13 +51,15 @@ Congratulations! You are now collecting logs using OpenTelemetry.
 
 What does the above program do?
 
-The program uses the
+The program creates a
```

Comment:
@cartersocha Please take a look. I think this can be further modified, similar to what you did for tracing doc. (code-snippet, along with explanation).